code_of_conduct.md

```diff
@@ -0,0 +1,180 @@
+- Continuing to initiate interaction with someone after being explicitly asked to stop
+- Offensive, insulting, derogatory, or degrading remarks
+- Demands for sexual favours in exchange for favourable or preferential treatment
+- Advocating for, or encouraging any of the above behaviours
+
+## Consequences of Unacceptable Behaviour
+Participants who are asked to stop any inappropriate behaviour are expected to comply immediately. This applies to any event or platform, either online or in-person. If an event participant engages in behaviour that violates this code of conduct, the organizers may warn the offender, ask them to leave the event or platform (without refund), or engage the Council to investigate the Code of Conduct violation and impose appropriate sanctions.
+
+## Incident reporting guidelines
+If you believe that someone is violating the Code of Conduct, please report this in a timely manner. Code of Conduct violations reduce the value of the community for everyone. The Council takes reports of misconduct very seriously and is committed to preserving and maintaining the welcoming nature of our community.
+
+### All reports will be kept confidential.
+
+Any violations to the Code of Conduct during Council events should be immediately reported to the event host, organiser or the designated incident person/people.
+
+- For the RSE Asia Australia Unconference please contact paula [dot] martinez [at] ardc.edu.au
+- For the any other situation, please contact the co-chair of the Steering Committee of the the RSE Association of Australia and New Zealand.
+
+If the violation occurs in an online space, reports should be emailed directly to the co-chair of the Council or any other Council member.
+
+The person to whom a report is made is hereafter referred to as "the Contact".
+
+All reports will be reviewed by the Council and will be kept confidential to the extent possible in a small group.
+
+In your report, please do your best to include:
+
+- Your contact information
+- Identifying information of the reported person
+- The behaviour associated with the Code of Conduct violation
+- If possible, where the Code of Conduct violation happened
+- The approximate time of the behaviour (if different than the time the report was made)
+- The circumstances surrounding the incident
+- Other people involved or that witnessed the incident
+- If there is a publicly available record (i.e. email thread, Slack messages, Twitter threads)
+
+If you believe this is an ongoing situation, please let us know
+- Any additional helpful information
+
+## What to do if someone is in immediate danger?
+
+If you believe someone is in immediate danger, please ask a Council member to contact appropriate emergency responders. All event organizers should, before the event, determine who would be appropriate to contact in case of an incident and make sure this information is available at all times over the duration of the event in question.
```

Comment:
```suggestion
If you believe someone is in immediate danger, please ask a Steering Committee member to contact appropriate emergency responders. All event organisers should, before the event, determine who would be appropriate to contact in case of an incident and make sure this information is available at all times over the duration of the event in question.
```